sound/soc/sof/intel/byt.c

```diff
@@ -455,6 +455,36 @@ static void byt_set_mach_params(const struct snd_soc_acpi_mach *mach,
 	mach_params->platform = dev_name(dev);
 }
 
+static int byt_reset_dsp_disable_int(struct snd_sof_dev *sdev)
```

Comment:
why return an int if the function can never fail?

Reply:
+1

Reply:
pushed a new branch with this change